tests/cpp/test_index_select.cpp

```diff
@@ -857,4 +857,51 @@ TEST_F(NVFuserTest, IndexSelectVectorization3DCase1) {
   testValidate(&fusion, outputs, {t0, t1}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, IndexPutAccumulate) {
```

Comment:
Could you please use `TEST_P` so that each configuration corresponds to a separate test case? That would be much easier to isolate failures when something only fails with some configuration.

Reply:
Also, this doesn't seem to fit well in this test file as it isn't `index_select`.

Reply:
Moved it to a separate file.